inc/views/layouts/layout_sidebar.php

```diff
@@ -75,54 +75,47 @@ private function render_sidebar_close( $slug ) {
 	 * @return array
 	 */
 	private function get_sidebar_setup( $context ) {
-		$sidebar_setup = array(
-			'theme_mod'    => 'neve_default_sidebar_layout',
+		$advanced_options = get_theme_mod( 'neve_advanced_layout_options', false );
+		$sidebar_setup    = array(
+			'theme_mod'    => '',
 			'sidebar_slug' => 'blog-sidebar',
 		);
-
-		if ( $context === 'blog-archive' ) {
-			$sidebar_setup['theme_mod'] = 'neve_blog_archive_sidebar_layout';
-
-			return $sidebar_setup;
-		}
-
-		if ( $context === 'single-post' ) {
-			if ( class_exists( 'WooCommerce' ) && is_product() ) {
-				$sidebar_setup['theme_mod']    = 'neve_single_product_sidebar_layout';
-				$sidebar_setup['sidebar_slug'] = 'shop-sidebar';
-
-				return $sidebar_setup;
+		if ( $advanced_options === false ) {
+			$sidebar_setup['theme_mod'] = 'neve_default_sidebar_layout';
+		} else {
```

Comment:
The `else` seems unnecessary here to me. You can just early return `$sidebar_setup` before the switch.